mtags-shared/src/main/scala/scala/meta/internal/metals/TimeFormatter.scala

```diff
@@ -0,0 +1,16 @@
+package scala.meta.internal.metals
+
+import java.text.SimpleDateFormat
+import java.util.Calendar
+
+import scala.util.Try
+import scala.util.matching.Regex
+
+object TimeFormatter {
+  private val format = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss-SSS")
+  val timeRegex: Regex =
+    "(\\d{4}-\\d{2}-\\d{2}_\\d{2}-\\d{2}-\\d{2}\\-\\d{3})".r
```

Comment:
This is highly annoying. It feels like there should be a way to get rid of this. 😂
I guess if you drop `_` from the pattern it can be used as a marker separating two parts or we could start w/ the date. All not very good ideas.

Reply:
I replaced `_` with `--`, so we don't need this regex